src/DotNetCore.CAP/DotNetCore.CAP.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.7.1" />
     <PackageReference Include="System.Threading.Channels" Version="4.7.1" />
+    <PackageReference Include="System.Text.Json" Version="4.7.2" />
```

Comment:
`System.Text.Json` dependency are not used, Are you planning to replace `Newtonsoft.Json` with `System.Text.Json`?

Reply:
I just removed the reference, it was a left over that slipped in the PR. At the moment I have no plans to replace `Newtonsoft.Json` with `System.Text.Json`...could be a future change ;-)